src/test/java/core/basesyntax/dao/impl/FruitStorageDaoImplTest.java

```diff
@@ -7,14 +7,14 @@
 import java.util.Set;
 import org.junit.After;
 import org.junit.Assert;
-import org.junit.BeforeClass;
+import org.junit.Before;
 import org.junit.Test;
 
 public class FruitStorageDaoImplTest {
-    private static FruitStorageDao dao;
+    private FruitStorageDao dao;
 
-    @BeforeClass
-    public static void beforeClass() {
+    @Before
+    public void setUp() {
```

Comment:
```suggestion
    @BeforeClass
    public static void setUp() {
```
you were supposed to change only method name, not annotation